exchanges/huobi/huobi_wrapper.go

```diff
@@ -1046,8 +1046,10 @@ func (h *HUOBI) SubmitOrder(ctx context.Context, s *order.Submit) (*order.Submit
 				oType = "optimal_20_ioc"
 			}
 		case order.Limit:
-			oType = "limit"
-			if s.TimeInForce.Is(order.PostOnly) {
+			switch {
+			case s.Type == order.Limit:
+				oType = "limit"
+			case s.TimeInForce.Is(order.PostOnly):
 				oType = "post_only"
```

Comment:
Why was this changed? It's nested in a case `order.Limit`. If I have my `order.Submit` struct Type as `order.Limit` and `TIF` as `order.PostOnly` to capture maker fees this won't work properly.